Pr 447

johannagonnzdz opened the pull request:
## What reference architecture does this PR apply to?


## Who should review your contribution? (Use @mention)


## Checklist before submitting
- [ ] My commits are only for the reference architecture mentioned above.
- [ ] I have followed the folder structure in the [main README](../README.md)

abhissharma21 commented on docusaurus.config.ts:
```diff
@@ -9,7 +9,7 @@ const config: Config = {
     tagline:
         'SAP Architecture Center empowers architects and developers with best practices, reference architectures, and community-driven guidance for designing, integrating, and optimizing SAP and cloud solutions. Accelerate innovation, ensure security, and reduce costs with proven frameworks and collaborative expertise for enterprise transformation.',
-    favicon: 'img/favicon.ico',
+    favicon: 'img/logo.png',
```
No, let me change it back

johannagonnzdz commented:
I reviewed the implementation and overall it looks good. I noticed a few adjustments are still needed, particularly with the margins in the technology domain section and the footer layout. In the technology domain section, the title isn’t clickable — I’m not sure if this is intentional. @cernus76, could you please check the functionality and confirm the expected behavior for these cards?

johannagonnzdz commented:
Hi all,
I just noticed that many classes have been removed from the custom.css file. I didn’t make these changes. Could this be due to the branch not being in sync, or @abhissharma21, were these modifications intentional?

This causes the footer to break in the mobile and tablet version. @navyakhurana

abhissharma21 commented on src/css/custom.css:
```diff
@@ -835,306 +833,14 @@ html[data-footer='false'] .footer {
     display: none;
 }
 
-/* Utility Classes */
```
It looks like custom.css  didn’t sync properly, so I’m recommitting it.

navyakhurana reviewed (commented):
Hi @johannagonnzdz @abhissharma21,
Thanks for the changes, it's already looking really good.
Few points from my side:

- The content in the banner is not properly displayed in the mobile version

<img width="446" height="408" alt="Screenshot 2025-08-25 at 11 58 46 AM" src="https://github.com/user-attachments/assets/06658411-e2c2-4e70-82db-5018275cda5e" />

- The community of practice picture is a bit blurred to me, is it for someone else too?
<img width="1614" height="475" alt="Screenshot 2025-08-25 at 12 01 13 PM" src="https://github.com/user-attachments/assets/2138f981-970f-464e-a09c-0c99bd00f7a9" />

- In the dark mode, the landing page needs some CSS fix with text being white, and background colour of the banner to match the site grey colour.
<img width="1695" height="857" alt="Screenshot 2025-08-25 at 12 29 56 PM" src="https://github.com/user-attachments/assets/425d1774-5b9b-4854-a564-563a6ea9cdc8" />

- Also, the background picture with the anvil needs to be added in the technology domain section (Please have a look at the mock-up for reference)

Can you fix these as well?

Regards,
Navya

johannagonnzdz commented:
I have fixed the banner in the mobile version. For the dark mode I haven't add the classes yet. Regarding the banners, these are not the final ones. I added the community of practice banner just for testing. PO is working on the banners.

cernus76 reviewed (approved):
Good to go!
